org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+{
+    "app-id": "org.siril.Siril",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "siril",
+    "rename-icon": "org.siril.Siril",
+    "rename-mime-file": "siril.xml",
+    "rename-mime-icons": [
+        "text-x-seq"
+    ],
+    "provides": {
+        "id": "org.free_astro.siril"
+    },
+    "replaces": {
+        "id": "org.free_astro.siril"
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
why is host needed?

Reply:
If I well remember, without it there are some permission issues.